docs/deployment/infrastructure.md

```diff
@@ -194,7 +194,12 @@ Use the following shorthand for conveying the Resource Type as part of the Resou
 The following steps are required to set up the environment, with linked issues to automate them:
 
 - `terraform apply`
+- Set up Slack notifications
+  1. [Create a Slack email](https://slack.com/help/articles/206819278-Send-emails-to-Slack) for the [#benefits-notify](https://cal-itp.slack.com/archives/C022HHSEE3F) channel
+  1. [Set it as a Secret in the Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/secrets/quick-create-portal#add-a-secret-to-key-vault) named `slack-benefits-notify-email`
```

Comment:
(minor formatting thing)

I noticed these ordered list items are showing up as just bullet points. 
I think indenting them with [four spaces](https://www.markdownguide.org/basic-syntax#adding-elements-in-lists) will get closer to what you intended.

*Currently*
![image](https://user-images.githubusercontent.com/25497886/205154254-854e9f1c-a389-4718-8459-36a5082a5d3f.png)

*Indented with four spaces*
![image](https://user-images.githubusercontent.com/25497886/205154336-7d23834e-f321-4add-8acf-ae75bc4b8557.png)

Reply:
Prettier forces it to two spaces 🙃 Not seeing a way to reconcile the two, so sidestepped the problem by removing the nested list.